catalog.html

```diff
@@ -0,0 +1,305 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>Поиск гостиницы в Седоне</title>
+
+
+  </head>
+<body>
+  <header class="page-header">
+    <nav class="main-navigation">
+      <ul class="navigation-list">
+        <li class="navigation-item">
+          <a class="navigation-link" href="index.html" aria-current="page">Главная</a>
+        </li>
+        <li class="navigation-item">
+          <a class="navigation-link" href="#">О Седоне</a>
+        </li>
+        <li class="navigation-item">
+          <a class="navigation-link navigation-link-current">Гостиницы</a>
+        </li>
+      </ul>
+      <ul>
+        <li>
+          <a href="catalog.html">
+            <img class="logo-header" src="" alt="Герб Седоны" >
+          </a>
```

Comment:
Ожидаемое поведение, что логотип в шапка указывает на главную
```suggestion
          <a href="index.html">
            <img class="logo-header" src="" alt="Герб Седоны" >
          </a>
```